form/unit_testing.rst

```diff
@@ -228,6 +228,10 @@ allows you to return a list of extensions to register::
         // ... your tests
     }
 
+.. deprecated:: 6.4
+
+    Annotations are deprecated since Symfony 6.4, use attributes instead.
```

Comment:
No, just remove the `addDefaultDoctrineAnnotationReader()` call from the example above.